example/timer/odp_timer_test.c

```diff
@@ -56,12 +56,12 @@ static const char *timerset2str(odp_timer_set_t val)
 	switch (val) {
 	case ODP_TIMER_SUCCESS:
 		return "success";
-	case ODP_TIMER_TOOEARLY:
```

Comment:
`ODP_TIMER_TOOEARLY` is still used in odp_timer_accuracy example.

Reply:
Fixed in v2